martinLi/assignments/treesBFS/lc637/lc637.js

```diff
@@ -0,0 +1,42 @@
+// Problem Statement #
+
+// Given a binary tree, populate an array to represent the averages of all of its levels.
```

Comment:
Was expecting a few lines of input/output, edges/corners, assumptions & high level overview